pom.xml

```diff
@@ -434,8 +451,19 @@
         <enabled>true</enabled>
       </snapshots>
     </repository>
+      <repository>
+          <id>jenkins-repo</id>
+          <url>https://repo.jenkins-ci.org/releases/</url>
+      </repository>
   </repositories>
 
+    <pluginRepositories>
+        <pluginRepository>
+            <id>jenkins-repo</id>
+            <url>https://repo.jenkins-ci.org/releases/</url>
+        </pluginRepository>
+    </pluginRepositories>
```

Comment:
Is this really needed? It looks like this dependency is available in Maven Central:
https://mvnrepository.com/artifact/org.jvnet.localizer/localizer/1.31

Reply:
I thought as much. But when I remove this pluginRepository, I get this error:

```
[ERROR] Failed to execute goal org.jvnet.localizer:localizer-maven-plugin:1.31:generate (default) on project openrefine: Execution default of goal org.jvnet.localizer:localizer-maven-plugin:1.31:generate failed: 
Plugin org.jvnet.localizer:localizer-maven-plugin:1.31 or one of its dependencies could not be resolved: 
Failed to collect dependencies at org.jvnet.localizer:localizer-maven-plugin:jar:1.31 -> org.kohsuke:access-modifier-annotation:jar:1.4 -> org.jenkins-ci:annotation-indexer:jar:1.4: Failed to read artifact descriptor for org.jenkins-ci:annotation-indexer:jar:1.4: org.jenkins-ci:jenkins:pom:1.26 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. 
This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced -> [Help 1]

```

Is there something I am missing out?

Reply:
Ok, of course that makes sense, I had not thought about transitive dependencies.